lib/ansible/plugins/connection/winrm.py

```diff
@@ -275,8 +275,8 @@ def exec_command(self, cmd, in_data=None, sudoable=True):
         except Exception:
             traceback.print_exc()
             raise AnsibleConnectionFailure("failed to exec cmd %s" % cmd)
-        result.std_out = to_bytes(result.std_out)
-        result.std_err = to_bytes(result.std_err)
+        result.std_out = result.std_out
+        result.std_err = result.std_err
```

Comment:
result.std_out and result.std_err is already byte at this point s and I don't think these need to be converted again.